tests/dummy/app/templates/application.hbs

```diff
@@ -12,4 +12,31 @@
     extra=(hash labelPath="name")
     as |user|}}
         {{user.name}}
+{{/power-select-infinity}}
+
+With Custom buffer size
+{{#power-select-infinity
+    options=options
+    search=(action 'search')
+    loadMore=(action 'loadMore')
+    selected=selected
+    onchange=(action (mut selected))
+    extra=(hash labelPath="name")
+    bufferSize=1
+    as |user|}}
+        {{user.name}}
+{{/power-select-infinity}}
+
+
+With Custom Loading Component
+{{#power-select-infinity
+    options=options
+    search=(action 'search')
+    loadMore=(action 'loadMore')
+    selected=selected
+    onchange=(action (mut selected))
+    extra=(hash labelPath="name")
+    loadingComponent=(component 'loading-component')
```

Comment:
@bakerac4 if the addon is using `{{component loadingComponent}}` I dont think you need to wrap the string in `(component )` here.